migration/src/main/resources/migrations/ensure-order-type-has-java_class_name-column.xml

```diff
@@ -5,49 +5,121 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 
-    <!-- In 1.10+, All users that created orders(orders.orderer) have provider accounts -->
 
+    <!-- Ensure order_type table has java_class_name and parent columns.  Credit k-joseph -->
     <changeSet id="ensure-order-type-has-java_class_name-column" author="IMB">
+        <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="order_type" columnName="java_class_name"/></not>
+            <not><columnExists tableName="order_type" columnName="parent"/></not>
+        </preConditions>
+        <comment>
+            Add java_class_name and parent columns to order_type table
+        </comment>
+        <addColumn tableName="order_type">
```

Comment:
I think best practice would probably be to put these in 2 different changesets, with the specific pre-condition, for each column, but this is likely fine.